README.md

```diff
@@ -1,141 +1,481 @@
+```
+
+### Configuration: MDAC/WDAC (Microsoft/Windows Data Access Components)
+
+TODO
+
+### Configuration: UnixODBC
+
+TODO
+
+### Configuration: iODBC
+
+TODO
+
+### Enabling driver manager tracing: MDAC/WDAC (Microsoft/Windows Data Access Components)
+
+Comprehensive explanations (possibly, with some irrelevant vendor-specific details though) on how to enable ODBC driver manager tracing could be found at the following links:
+
+- [5.8.1 Enabling ODBC Tracing on Windows](https://dev.mysql.com/doc/connector-odbc/en/connector-odbc-configuration-trace-windows.html)
+- [ODBC Troubleshooting: How to Enable Driver-manager Tracing](https://www.simba.com/blog/odbc-troubleshooting-tracing/)
+- [Enabling Tracing](https://docs.microsoft.com/en-us/sql/odbc/reference/develop-app/enabling-tracing)
+
+### Enabling driver manager tracing: UnixODBC
+
+Comprehensive explanations (possibly, with some irrelevant vendor-specific details though) on how to enable ODBC driver manager tracing could be found at the following links:
+
+- [ODBC Troubleshooting: How to Enable Driver-manager Tracing](https://www.simba.com/blog/odbc-troubleshooting-tracing/)
+- [How do I turn unixODBC tracing on or off?](https://www.easysoft.com/support/kb/kb00945.html)
+
+### Enabling driver manager tracing: iODBC
+
+Comprehensive explanations (possibly, with some irrelevant vendor-specific details though) on how to enable ODBC driver manager tracing could be found at the following links:
+
+- [ODBC Troubleshooting: How to Enable Driver-manager Tracing](https://www.simba.com/blog/odbc-troubleshooting-tracing/)
+- [Tracing Application Behavior](http://www.iodbc.org/dataspace/doc/iodbc/wiki/iodbcWiki/FAQ#Tracing%20Application%20Behavior)
+
+### Building from sources: Windows
+
+#### Build-time dependencies <!-- omit in toc -->
+
+CMake bundled with the recent versions of Visual Studio can be used.
+
+An SDK required for building the ODBC driver is included in Windows SDK, which in its turn is also bundled with Visual Studio.
+
+You will need to install WiX toolset to be able to generate `.msi` packages. You can download and install it from [WiX toolset home page](https://wixtoolset.org/).
+
+#### Build steps <!-- omit in toc -->
+
+All of the following commands have to be issued in Visual Studio Command Prompt:
+
+- use `x86 Native Tools Command Prompt for VS 2019` or equivalent for 32-bit builds
+- use `x64 Native Tools Command Prompt for VS 2019` or equivalent for 64-bit builds
+
+Clone the repo with submodules:
+
+```sh
+git clone --recurse-submodules git@github.com:ClickHouse/clickhouse-odbc.git
+```
+
+Enter the cloned source tree, create a temporary build folder, and generate the solution and project files in it:
+
+```sh
+cd clickhouse-odbc
+mkdir build
+cd build
+
+# Configuration options for the project can be specified in the next command in a form of '-Dopt=val'
+
+# Use the following command for 32-bit build only.
+cmake -A Win32 -DCMAKE_BUILD_TYPE=RelWithDebInfo ..
```

Comment:
More for demonstration purposes. I'll remove that.

Reply:
Actually, for Xcode and VS solutions, if you are building from console by calling `cmake build . ...` you have to specify build type explicitly, since the generated solutions are multi-build.
So keeping this in mind, it is always a good idea to always have build type specified explicitly in such commands, even though for demonstration purposes.

Reply:
I meant `-DCMAKE_BUILD_TYPE=RelWithDebInfo` (which should be used by default), not a `-A Win32`

Reply:
When you build using `cmake --build .` you need to provide `-DCMAKE_BUILD_TYPE=RelWithDebInfo` in Xcode or VS generated projects.